### PR TITLE
Expanded the usage instructions.

### PR DESCRIPTION
**Pull request by odisseus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,43 @@
...
+    * JDK 11 should work as well. 
+    * Other versions, including newer ones, are not supported.
+* Bloop
+    * If Fury cannot detect a running Bloop server, it will download and launch an own instance.
+* Git
+* IPFS
+    * If Fury cannot detect a running IPFS daemon, a new one will be started. 
+    * Fury will also download the IPFS distribution if an installed one cannot be found.
+* Tab completion for shell commands works only in [Fish](https://fishshell.com/). 
+    * Support for Bash and Zsh may appear in the future.
```
Comment (propensive) [inline README.md]: You need to swap `fish` and `zsh`.

File: README.md [hunk 2]
```diff
@@ -2,19 +2,43 @@
...
+
+#### Checking if it works
+* Run the integration tests: `etc/integration`
+* Build Fury itself: `fury`
+
+### Website
+Full information about Fury is available on the [Fury website](https://fury.build/).
+
+### Status
+Fury is under active development, but a binary release is not publicly
```
Comment (propensive) [inline README.md]: I'm still working towards a release on Wednesday, though. ;)

File: README.md (hunk 1 above)
Comment (odisseus) [reply README.md]: OK

File: README.md (hunk 2 above)
Comment (odisseus) [reply README.md]: Ok, I'll remove this.

**Review by propensive — APPROVED** [review]

**Pull request merged**